src/server/pfs/fuse/server.go

```diff
@@ -124,12 +124,13 @@ type Response struct {
 }
 
 type DatumState struct {
-	Datums              []*pps.DatumInfo
-	PaginationMarker    string
-	DatumInput          *pps.Input
-	DatumInputsToMounts map[string][]string
-	DatumIdx            int
-	AllDatumsReceived   bool
+	Datums            []*pps.DatumInfo
+	PaginationMarker  string
+	DatumInput        *pps.Input
+	SimpleInput       bool
```

Comment:
```suggestion
	// SimpleInput tracks whether 'DatumInput' is simple (i.e. not a group by or
	// join). This is set by 'parseInput', and later read by 'GetDatums' to
	// decide whether to list datums using Pachyderm's 'GlobFile' or 'ListDatum'
	// call
	SimpleInput       bool
```

I'm not 100% sure this description is right, but I wanted to include some kind of comment (I know the other fields aren't commented but at least it's a start). I basically just wanted to propose documenting: where it's set, where it's read, and what it's used for.